packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -33,17 +130,17 @@
 }
 
 a {
+  font-weight: 500 !important;
```

Comment:
Is the `!important` declaration necessary here?

Reply:
no need